lighthouse-core/audits/byte-efficiency/uses-responsive-images.js

```diff
@@ -41,9 +41,9 @@ class UsesResponsiveImages extends ByteEfficiencyAudit {
       description: 'Properly size images',
       informative: true,
       helpText:
-        'Image sizes served should be based on the device display size to save network bytes. ' +
-        'Learn more about [responsive images](https://developers.google.com/web/fundamentals/design-and-ui/media/images) ' +
-        'and [client hints](https://developers.google.com/web/updates/2015/09/automating-resource-selection-with-client-hints).',
+        'Serve images that are smaller than the user\'s viewport to save cellular data ' +
```

Comment:
P.S. that's not a nit, that's a significant change in behavior

Somebody might fail the audit, read the description I wrote, and be like "wtf why am I failing this, the image is not larger than the viewport"